digitalocean/app/app_spec.go

```diff
@@ -875,6 +875,52 @@ func appSpecLogDestinations() *schema.Resource {
 				Required:    true,
 				Description: "Name of the log destination",
 			},
+			"open_search": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				MaxItems:    1,
+				Description: "OpenSearch configuration.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"endpoint": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "OpenSearch endpoint.",
+						},
+						"basic_auth": {
+							Type:        schema.TypeList,
+							Required:    true,
+							Description: "Basic authentication details.",
+							MaxItems:    1,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"user": {
+										Type:        schema.TypeString,
+										Required:    true,
```

Comment:
```suggestion
										Optional:    true,
```
This is also optional:

> Username to authenticate with. Only required when endpoint is set. Defaults to doadmin when cluster_name is set.